go.mod

```
@@ -47,6 +47,7 @@ require (
 	k8s.io/apimachinery v0.23.3
 	k8s.io/client-go v0.23.3
 	k8s.io/component-helpers v0.23.3
+	k8s.io/klog v1.0.0
```

```suggestion
	k8s.io/klog/v2 v1.0.0
```

it should already be imported, fix the imports in the files